app/Commands/ReposCommand.php

```
@@ -0,0 +1,414 @@
+        table($headers, $rows);
+        return 0;
+    }
+
+    private function displayInteractive(): int
+    {
+        info("📁 Found " . count($this->allRepos) . " repositories");
+        
+        while (true) {
+            // Use search for better UX with large repo lists
+            $repoOptions = [];
+            foreach ($this->allRepos as $index => $repo) {
```

PHPMD (2.15.0)

Avoid unused local variables such as '$index'. (Unused Code Rules)

(UnusedLocalVariable)